README.md

```diff
@@ -42,14 +42,16 @@ The install generator does the following
 * Adds a line in route for mounting.
 
 Generator that installs filter and table files to application
-
+    $ rails g spotlight_search filter orders --filters scope_name:filter_type
     $ rails g spotlight_search:filter orders --filters search:input order_status:multi_select status:select
```

Comment:
why are we having this line bro?

Reply:
First line is syntax, Second line is example

Reply:
ohh okay

Reply:
but it has an extra colon in it